tasks/cat2.yml

```diff
@@ -434,9 +434,27 @@
 # - name: V-38673 Medium  The operating system must ensure unauthorized, security-relevant configuration changes detected are tracked
 # - name: V-38670 Medium  The operating system must detect unauthorized changes to software and information
 
-- name: V-38671 Medium  The sendmail package must be removed
-  yum: name=sendmail state=absent
-  tags: [ 'cat2' , 'V-38671' , 'sendmail' , 'unauthorized_packages' ]
+- name: "| MEDIUM | V-38671 | PATCH |\n
```

Comment:
On these next two tasks can you fix your indentation to 4 spaces for the module args and tags? Also, feel free to start doing the one line names.